app/src/main/java/com/github/gotify/service/WebSocketConnection.java

```diff
@@ -141,6 +173,11 @@ public void onClosed(WebSocket webSocket, int code, String reason) {
             super.onClosed(webSocket, code, reason);
         }
 
+        synchronized void scheduleReconnect(long delay, int reason) {
+            WebSocketConnection.this.scheduleReconnect(delay, reason);
+            onReconnectSchedule.execute(delay, reason);
```

Comment:
This is already executed in WebSocketConnection.this.scheduleReconnect, so this method can be removed completely.